### ISSUE TITLE
Courier New for code,pre looks awful in Ubuntu Linux 15.10

### ISSUE THREAD

**Issue by gjcarneiro** [twbs/bootstrap#18609]
Please don't put  "Courier New" ahead of monospace:

```css
code,
pre {
  font-family: Monaco, Menlo, Consolas, "Courier New", monospace;
}
```

This causes Courier New to be used in Linux and it looks really awful.  I don't understand the need for overriding the default monospace font.  The default monospace font is usually the most readable on any system, trying to override is a very bad idea.

**Comment by kkirsche** [comment]
Do you have clear examples of the impact to readability you mention?

**Comment by cvrebert** [comment]
I think "Linux" might be too wide a category to usefully generalize about here.
@gjcarneiro What distro are you running? What package does your copy of Courier New come from?

**Comment by gjcarneiro** [comment]
Ubuntu 15.10.  The fonts are installed by the package `ttf-mscorefonts-installer`.

**Comment by gjcarneiro** [comment]
One example of a pre style that works well in Linux is the one here, in github, which is:

```css
    font: 12px Consolas, "Liberation Mono", Menlo, Courier, monospace;
```

Apparently, "Liberation Mono" is the name of a font I have installed here and which displays nice.

@kkirsche my main gripe is that the Courier New font renders in too thin strokes.  The lines that for the letters are so thin that makes them too hard to read on a 1920x1200 24" display.

Perhaps one way to fix it is to do similarly to github and add ""Liberation Mono" to the list of font families, making sure it comes before "Courier New".

**Comment by kkirsche** [comment]
Personally I admit I'm a fan of consolas / liberation mono over courier new, wanted to better understand your opinion though. Thanks for addressing my question :)

**Comment by cvrebert** [comment]
@gjcarneiro Could you post a screenshot of how Courier New renders for you?

**Comment by gjcarneiro** [comment]
Here's with Courier New:

![couriernew](https://cloud.githubusercontent.com/assets/2197096/11914324/58cdab44-a675-11e5-8a35-f57dbfbf6acb.png)

And here's with Liberation Mono (much better):

![liberationmono](https://cloud.githubusercontent.com/assets/2197096/11914323/58cacc76-a675-11e5-8ac9-97f8d4ec0926.png)

**Comment by cvrebert** [comment]
X-Ref: #16225

**Comment by cvrebert** [comment]
Fixed by #18814.
